vector/src/test/java/im/vector/app/features/settings/devices/v2/GetDeviceFullInfoListUseCaseTest.kt

```diff
@@ -110,21 +117,24 @@ class GetDeviceFullInfoListUseCaseTest {
                 cryptoDeviceInfo = cryptoDeviceInfo1,
                 roomEncryptionTrustLevel = RoomEncryptionTrustLevel.Trusted,
                 isInactive = true,
-                isCurrentDevice = true
+                isCurrentDevice = true,
+                deviceUserAgent = DeviceUserAgent(DeviceType.MOBILE)
```

Comment:
This test should not pass normally. We expect to have the mocked result as `deviceUseAgent` value:
```
DeviceUserAgent(
                DeviceType.MOBILE,
                "Xiaomi Mi 9T",
                "Android 11",
                "Element dbg",
                "1.5.0-dev"
        )
```
This means the assertion seems to be not correct in a certain way since it should not be equal to `DeviceUserAgent(DeviceType.MOBILE)`.

Reply:
It is because filter use case is a mockk and returning expected result. Anyway, we don't need to test it, removed.